examples/mysql-wordpress-pd/README.md

```diff
@@ -336,27 +266,33 @@ http://<external-ip>
 
 You should see the familiar WordPress init page.
 
+![WordPress init page](WordPress.png "WordPress init page")
+
 ## Take down and restart your blog
 
-Set up your WordPress blog and play around with it a bit.  Then, take down its pods and bring them back up again. Because you used persistent disks, your blog state will be preserved.
+Set up your WordPress blog and play around with it a bit. Then, take
+down its pods and bring them back up again. Because you used
+persistent disks, your blog state will be preserved.
 
-If you are just experimenting, you can take down and bring up only the pods:
+All of the resources are labeled with `app=wordpress`, so you can
+easily bring them down with:
 
-```sh
-$ kubectl delete -f examples/mysql-wordpress-pd/wordpress.yaml
-$ kubectl delete -f examples/mysql-wordpress-pd/mysql.yaml
+```shell
+kubectl delete deployment,pod,service -l app=wordpress
```

Comment:
Presumably `kubectl delete deployment` deletes the pods it controls cascadingly.